src/main/java/org/hisp/quick/statementbuilder/MySqlStatementBuilder.java

```diff
@@ -98,4 +98,29 @@ public String getDoubleColumnType()
     {
         return "double";
     }
+
+    @Override
+    public String getUpsertStatement( T object )
+    {
+        StringBuffer buffer = new StringBuffer();

```

Comment:
As sonar pointed out better use `StringBuilder`

Reply:
Changed this to StringBuilder.